Makefile

```diff
@@ -213,7 +213,7 @@ $(BUILT_D3):
 	  node_modules/d3/src/svg/line.js \
 	  node_modules/d3/src/svg/arc.js \
 	  node_modules/d3/src/event/drag.js \
-	  $(GUISRC)/app/assets/javascripts/unscaled-pack.js \
+		node_modules/d3/src/layout/pack.js \
```

Comment:
We will need to qa this once we have a charm installable GUI again to make sure that this functionality is either a) good enough or b) we have to write a new unscaled-pack. The methods our unscaled-pack relies on are no longer available in the latest version of d3